Dijkstra's Single-Source Shortest Path/Dijkstra.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream> 
+#include <limits.h> 
+
+// Define Number of vertices in the graph 
+#define V 7
```

Comment:
It is not a good practice to use macros to define constants.
Refer [this](https://rules.sonarsource.com/cpp/tag/bad-practice/RSPEC-5028) SONAR rule.

Reply:
i have made the changes and issued a new pull request